dashboard/config/upstream-kasan.config

```diff
@@ -6675,7 +6682,7 @@ CONFIG_DEBUG_INFO=y
 # CONFIG_DEBUG_INFO_COMPRESSED is not set
 # CONFIG_DEBUG_INFO_SPLIT is not set
 # CONFIG_DEBUG_INFO_DWARF4 is not set
-# CONFIG_DEBUG_INFO_BTF is not set
+CONFIG_DEBUG_INFO_BTF=y
```

Comment:
Oh, I forgot about this. Unfortunately, this cannot be enabled:
https://github.com/google/syzkaller/blob/master/dashboard/config/bits-syzbot.config#L153

Reply:
Ok, that will be a bad news because BPF LSM requires the existence of vmlinux. Even if we replace the hook id with some random integer, it will never pass the verification ([see here](https://elixir.bootlin.com/linux/latest/source/kernel/bpf/verifier.c#L10668)).

Is it possible to update pahole somehow?

Reply:
The ideal way to update it would be update the distro-provided package. But I just tried, and it's still not there:
```
# apt-get upgrade dwarves
# pahole --version
v1.9
```

Updating images on all syzbot machines is not something I can take on right now.
I have not looked into compiling a fresh version of pahole from source.